mkosi/machine.py

```diff
@@ -88,6 +89,7 @@ def __enter__(self) -> Machine:
             init_namespace(self.args)
             build_stuff(self.args)
 
+    def __boot_image(self) -> None:
```

Comment:
Private methods in python should generally be prefixed with just a single underscore, not two.